pkg/volume/plugins.go

```diff
@@ -131,26 +131,26 @@ type VolumePluginMgr struct {
 
 // Spec is an internal representation of a volume.  All API volume types translate to Spec.
 type Spec struct {
-	Name                   string
-	VolumeSource           api.VolumeSource
-	PersistentVolumeSource api.PersistentVolumeSource
-	ReadOnly               bool
+	Name             string
+	Volume           *api.Volume
```

Comment:
Add a comment that, while Volume and PersistentVolume have a Name field, that is ignored.

Or should we EOL Spec.Name instead?